src/misc/ArborX_SymmetricSVD.hpp

```diff
@@ -77,8 +77,63 @@ KOKKOS_FUNCTION auto argmaxUpperTriangle(Matrix const &mat)
   return result;
 }
 
+// Function to perform a single Jacobi rotation
+template <typename Matrix, typename Unit>
+KOKKOS_FUNCTION void jacobi_rotate(Matrix &A, Unit &V, int p, int q)
```

Comment:
Looks good but you might want to spell out type requirements (Matrix and Unit are View-like and rank-2) and preconditions (A and V are both n-by-n,  and 0 <= p,q < n
You could do it via a static assert or a requires clause, and a few KOKKOS_ASSERT()